vantage6-server/vantage6/server/__init__.py

```
@@ -717,13 +720,16 @@ def load_resources(self) -> None:
             "storage_adapter": self.storage_adapter,
             **services
         }
-
         for res in RESOURCES:
-            module = importlib.import_module("vantage6.server.resource." + res)
-            if res in ["result"]:
-                module.setup(self.api, self.ctx.config["api_path"], result_services)
-            else:
-                module.setup(self.api, self.ctx.config["api_path"], services)
+            try:
+                module = importlib.import_module("vantage6.server.resource." + res)
+                if res in ["result"]:
+                    module.setup(self.api, self.ctx.config["api_path"], result_services)
+                else:
+                    module.setup(self.api, self.ctx.config["api_path"], services)
+            except Exception as e:
```

if this fails while large results are configured we should probably crash/fail early here? Rather than finding out during a run. In general I'm not very happy with how I set this up, making this exceptional case for when there is a result resource.